docs/source/guides/user/chapters/plugins.rst

```diff
@@ -175,10 +175,10 @@ test phases.  Here's a sequence of the job phases:
 Plugin writers can have their own code called at Avocado during a job
 by writing a that will be called at phase number 2 (``pre_tests``) by
 writing a method according to the
-:meth:`avocado.core.plugin_interfaces.JobPreTests` interface.
+:meth:`avocado.core.plugin_interfaces.PreTest` interface.
```

Comment:
I don't think this change is accurate... the `JobPreTests` interface is called by a job *once*, before any test is run.  The newly introduced interface is about creating runnables that will run before each test runnable, right?

It's OK to rethink the validity of `JobPreTests` (and `JobPostTests`) but they are not interchangeable.

Reply:
Yes, you are right, this is mistaken. IMO the `JobPreTests` and `JobPostTests` are still valid, but we need to incorporate them into dependency graph, so that they will be available also for different spawners.